WordPress/src/test/java/org/wordpress/android/usecase/social/GetJetpackSocialShareLimitStatusUseCaseTest.kt

```diff
@@ -95,4 +96,23 @@ class GetJetpackSocialShareLimitStatusUseCaseTest : BaseUnitTest() {
         val actual = classToTest.execute(siteModel)
         assertEquals(expected, actual)
     }
+
+    @Test
+    fun `Should return Disabled if fetch site fails`() = test {
+        val siteModel = SiteModel().apply {
+            siteId = 1L
+            setIsJetpackInstalled(true)
+            planActiveFeatures = ""
+        }
+        whenever(siteStore.fetchJetpackSocial(siteModel)).thenReturn(
+            FetchedJetpackSocialResult.Error(
+                SiteStore.SiteError(
+                    SiteStore.SiteErrorType.GENERIC_ERROR
+                )
+            )
+        )
+        val expected = ShareLimit.Disabled
+        val actual = classToTest.execute(siteModel)
```

Comment:
Good question. I'm not aware of any internal agreement about naming the instance of the class being tested. I've been using `classToTest` for some time, and almost every unit test class I wrote from scratch in this project is using this standard (from what I've seen it's being used in 41 files so far).

The reason I like this naming approach is exactly what you said: it's easy for someone without context to check any specific test function and immediately identify the instance being tested. Besides, it creates some standard and what to expect when opening a unit test file. The standard it creates is that whenever there's a `classToTest` doing something, the developer will know that it is the `Act` part considering the `Arrange, Act, Assert` pattern, and understand that we should be asserting something after that. This naming which makes the `Act` part easier to identify plus the usage of `actual` and `expected` in the context of `Assert` IMHO makes it a lot easy to understand a test specially when we have a lot of dependencies and mocks.

Another thing to notice is that usually when someone opens a class test is because they've opened the real class before, so IMHO they might already have a good amount of context when opening it and usually can just focus in understanding what's being tested and what's expected in a certain situation.

In your opinion it makes more sense to use the name of the class instead?